TranslatorReasonerAPI.yaml

```
@@ -364,20 +364,16 @@ components:
         of the QNode will be returned in the Results.
       properties:
         id:
-          oneOf:
-            - $ref: '#/components/schemas/CURIE'
-            - type: array
-              items:
-                $ref: '#/components/schemas/CURIE'
-          example: OMIM:603903
+          type: array
+          items:
+            $ref: '#/components/schemas/CURIE'
+            example: OMIM:603903
```

```suggestion
          example: ['OMIM:603903']
```
The `$ref` technically overrides this `example`, and I think it's clearer to have an example of the `id` value itself.

---

Thanks for the suggestion. I completely agree. This took me a few times to pass tests, but at least I now have a local dev environment set up for working on the TRAPI!